include/net/lwm2m_client_utils.h

```diff
@@ -68,6 +68,42 @@ struct modem_mode_change {
  */
 int lwm2m_init_security(struct lwm2m_ctx *ctx, char *endpoint, struct modem_mode_change *mmode);
 
+/**
+ * @brief Set security object to PSK mode.
+ *
+ * Any pointer can be given as a NULL, which means that data related to this field is set to
+ * zero legth in the engine. Effectively, it causes that relative data is not written into a modem.
+ * This can be used if the given data is already provisioned to modem.
+ *
+ * @param sec_obj_inst Security object ID to modify.
+ * @param psk Pointer to PSK key, either in hex or binary form.
+ * @param psk_len Length of data in psk pointer.
+ * @param psk_is_hex True if psk points to data in HEX format. False if the data is binary.
+ * @param psk_id PSK key id in string format.
+ * @return int zero if success, negative error code otherwise.
```

Comment:
```suggestion
 * @return Int zero if success, negative error code otherwise.
```

Reply:
Actually.. I'll drop the `int` from there. Looks like my editor autocompleted those, but don't use return type names in documentation.
So it will be `@return Zero if succes, ...`